### ISSUE TITLE
AndroidManifest changes getting reverted by gradle

### ISSUE THREAD

**Issue by dilipdodiya** [jamesmontemagno/vsts-mobile-tasks#17]
First of all I would like to thank you for wonderful VSTS Task. I am using VSTS task - ‘Mobile App Tasks for iOS and Android’ developed by you. This task is really very helpful.

It's working very well for iOS build but there is problem with Android build. It seems changes to ‘AndroidManifest’ file getting reverted by gradle build process ☹. So in case of Android build, this task should update ‘build.gradle’ file instead of ‘AndroidManifest’.

**Comment by jamesmontemagno** [comment]
Yes, I don't support Gradle at this time. You can make it so Gradle doesn't override tour manifest. I will look to implement this in the future.

**Comment by nkovacic** [comment]
Having the same issue as our project has only in build.gradle versionCode and versionName.
Are there any plans in the foreseeable future to implement this feature?

**Comment by cbnewham** [comment]
Yep. Unfortunately this is useless because it doesn't support the standard way to build Android projects via Gradle. It seems this has become abandonware...  :-(

**Comment by jamesmontemagno** [comment]
Submit a PR, i'm not a gradle expert

**Comment by BMuuN** [comment]
I'm no Android or gradle expert but I just moved the `versionCode` and `versionName` properties from the gradle file into the `AndroidManifest.xml` and this extension works perfectly.  Coupled with the [Google Play](https://marketplace.visualstudio.com/items?itemName=ms-vsclient.google-play) extension this task work perfectly for pushing new APK's into the Play Store.

Thanks for your efforts, check your [buymeacoffee.com](https://www.buymeacoffee.com/jamesmontemagno) account, have a coffee on me 👍

**Comment by tyagirajnish** [comment]
facing the same issue, i have already moved my versionCode and versionName to androidManifest.xml but no luck :(
